datasets/dataset_setup.py

```diff
@@ -469,15 +458,15 @@ def download_librispeech(dataset_dir, tmp_dir, train_tokenizer):
 
     # Preprocess data.
     tokenizer_vocab_path = os.path.join(tmp_librispeech_dir, 'spm_model.vocab')
-    librispeech_dir = os.path.join(dataset_dir, 'criteo')
+    librispeech_dir = os.path.join(dataset_dir, 'librispeech')
     librispeech_preprocess.run(
         input_dir=tmp_librispeech_dir,
         output_dir=librispeech_dir,
         tokenizer_vocab_path=tokenizer_vocab_path)
 
 
-def download_ogbg(data_dir, tmp_dir):
-  pass
+def download_ogbg(data_dir):
+  tfds.builder('ogbg_molpcba:0.1.3', data_dir=data_dir).download_and_prepare()
```

Comment:
I get the following error : AssertionError: Failed to construct dataset ogbg_molpcba: Dataset ogbg_molpcba cannot be loaded at version 0.1.3, only: 0.1.2.

Reply:
I fixed it by reverting to 0.1.2.